modules/gapi/misc/python/test/test_gapi_streaming.py

```diff
@@ -229,7 +229,6 @@ def test_video_good_features_to_track(self):
 
 
         def test_gapi_streaming_meta(self):
-            ksize = 3
```

Comment:
Is it necessary?

Reply:
This `ksize` isn't used in this particular test, I've just noticed that and decided to clear that